es-ds-docs/pages/molecules/progress-circle.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <div>
+        <h1>
+            Progress circle
+        </h1>
+        <p class="pb-200">
+            Extended from <b-link
+                href="https://github.com/EnergySage/es-cdgm/blob/main/frontend/components/CircularProgress.vue"
+                target="_blank">
+                <code>es-cdgm</code> codebase
+            </b-link>
+        </p>
+        <es-progress-circle
+            class="mb-100"
+            :value="value"
+            height="100px" />
+        <div class="d-flex flex-nowrap">
+            <es-button
```

Comment:
For the time being, we should leave out references to es-button, since it's not available and causes a hydration error. The followup work is captured here: https://energysage.atlassian.net/browse/CED-1818